.pre-commit-config.yaml

```diff
@@ -12,6 +12,7 @@ repos:
         -   --no-puppet_url_without_modules-check
         -   --no-arrow_alignment-check
         -   --no-arrow_on_right_operand_line-check
+        -   --no-variable_is_lowercase-check
```

Comment:
Unfortunately since `$::dnsA`, `$::dnsCname`, and `$::ipHostNumber` come from LDAP, there's not much we can do here except maybe modify the [ldap-enc script](https://github.com/ocf/puppet/blob/master/modules/ocf_puppet/files/ldap-enc) to lowercase them or something.